src/ListBox/README.md

```diff
@@ -0,0 +1,132 @@
+# ListBox
+
+Provides a ListBox component that can takes Option components as children and handles selection of and navigation through the Options.
+
+## API
+
+### `children` (Option component array; required)
+`children` specifies the list of Option components that will be displayed within the ListBox element.
+
+### `className` (string array; optional)
+`className` specifies Bootstrap class names to apply to the ListBox component. The default is an empty array.
+
+### `selectedOption` (?)
```

Comment:
I'm not sure what the type should be. An Option? The index of the Option?

Reply:
The index would be easiest for whom? The component or the user? Or both? The one hang up I have about the index is that I'm not sure how to allow the user to "clear" the selection -1? null? If a user were to pass in an actual Option, that would be a little easier to do (i.e. null or not in the children array). For some reason, it also feels a little more intuitive for me to say, "I want this Option selected" as opposed to "I want the Option at index x selected", but I guess then they'd have to keep a reference to the Option element and then we'd need to do some deep equality checking, so perhaps the index is the way to go.

Reply:
I will check on the question of the props and report back.

Reply:
Re: the index being easier, yeah I think of it like a primary key in a database. It's easier to check for equality against and you don't have to store the whole element in your redux store if you have one.

I think selectedOption as an override, but still allowing user selection to override *that* makes the most sense :+1: . I just think we should document that behavior here for clarity.